dcmgr/lib/dcmgr/node_modules/maintenance.rb

```diff
@@ -7,12 +7,14 @@ class Maintenance < Isono::NodeModules::Base
       @thread_pool = Isono::ThreadPool.new(1, 'Maintenance')
       @thread_pool.pass {
         myinstance.dup.expire_ip_handles
+        myinstance.dup.automatic_windows_password_deletion
```

Comment:
I wondered the same thing.  Copying Jari's code example, which apparently has been working OK.